.github/workflows/btr-ui-ci.yml

```diff
@@ -126,7 +130,7 @@ jobs:
         uses: cypress-io/github-action@v6
         with:
           working-directory: btr-web/${{ matrix.project }}
-          build: pnpm run build
+          build: pnpm run build:local
```

Comment:
nuxt generate this isn't building properly here and causes errors with cypress unless you only visit('/'). Changed to nuxt build instead of generate here for now until we figure that out (might get solved by upgrading nuxt)